snap/snapcraft.yaml

```diff
@@ -28,136 +32,28 @@ apps:
-    plugin: nil
-    override-pull: |
-      cat > alsa.conf <<EOF
-      pcm.!default {
-        type pulse
-        fallback "sysdefault"
-        hint {
-          show on
-          description "Default ALSA Output (currently PulseAudio Sound Server)"
-        }
-      }
-      ctl.!default {
-        type pulse
-        fallback "sysdefault"
-      }
-      EOF
-    override-build: |
-      snapcraftctl build
-      install -m644 -D -t $SNAPCRAFT_PART_INSTALL/etc alsa.conf
-    build-packages:
-    - libasound2-dev
-    stage-packages:
-    - libasound2
-    - libasound2-plugins
-
-  desktop-glib-only:
-    source: https://github.com/ubuntu/snapcraft-desktop-helpers.git
-    source-subdir: glib-only
-    plugin: make
-    build-packages:
-    - libglib2.0-dev
-    stage-packages:
-    - libglib2.0-bin
-
-  freetype-prebuild:
-    source: https://download.savannah.gnu.org/releases/freetype/freetype-2.11.0.tar.xz
-    source-checksum: sha256/8bee39bd3968c4804b70614a0a3ad597299ad0e824bc8aad5ce8aaf48067bde7
-    plugin: autotools
-    configflags:
-    - --prefix=/usr/local
-    - --with-harfbuzz=no
-    build-packages:
-    - libbz2-dev
-    - libpng-dev
-    - zlib1g-dev
-    stage-packages:
-    - libbz2-1.0
-    - libpng16-16
-    - zlib1g
-    prime:
-    - -*
-  
-  freetype:
-    after: [freetype-prebuild, harfbuzz]
-    source: https://download.savannah.gnu.org/releases/freetype/freetype-2.11.0.tar.xz
-    source-checksum: sha256/8bee39bd3968c4804b70614a0a3ad597299ad0e824bc8aad5ce8aaf48067bde7
-    plugin: autotools
-    configflags:
-    - --prefix=/usr
-    - --with-harfbuzz=yes
-    build-environment:
-    - CFLAGS: -O2 -pipe -I$SNAPCRAFT_STAGE/usr/include/harfbuzz
-    - CXXFLAGS: -O2 -pipe -I$SNAPCRAFT_STAGE/usr/include/harfbuzz
-    build-packages:
-    - libbz2-dev
-    - libpng-dev
-    - zlib1g-dev
-    stage-packages:
-    - libbz2-1.0
-    - libpng16-16
-    - zlib1g
-
-  harfbuzz:
-    after: [freetype-prebuild]
-    source: https://www.freedesktop.org/software/harfbuzz/release/harfbuzz-2.6.7.tar.xz
-    source-checksum: sha256/49e481d06cdff97bf68d99fa26bdf785331f411614485d892ea4c78eb479b218
-    plugin: autotools
-    configflags:
-    - --prefix=/usr
-    - --with-freetype
-    build-environment:
-    - CFLAGS: -O2 -pipe -I$SNAPCRAFT_STAGE/usr/local/include/freetype2
-    - CXXFLAGS: -O2 -pipe -I$SNAPCRAFT_STAGE/usr/local/include/freetype2
-    - LDFLAGS: -L$SNAPCRAFT_STAGE/usr/lib -L$SNAPCRAFT_STAGE/usr/local/lib
-    build-packages:
-    - g++
-    - gperf
-    - libglib2.0-dev
-    - libgraphite2-dev
-    - uuid-dev
-    stage-packages:
-    - libglib2.0-bin
-    - libgraphite2-3
-    - libpng16-16
-    - libuuid1
-
   supertuxkart:
-    after:
-    - freetype
-    - harfbuzz
     plugin: cmake
     source: https://github.com/supertuxkart/stk-code/releases/download/$SNAPCRAFT_PROJECT_VERSION/SuperTuxKart-$SNAPCRAFT_PROJECT_VERSION-src.tar.xz
     override-pull: |
       snapcraftctl pull
```

Comment:
it looks like this is the line causing the issue in the build service - it's saying we need to use craftctl default.

I don't get why that is only affecting the build service tho, and then only the non-x86_64 build, when my local build on ARM64 worked just fine.

Reply:
![image](https://github.com/diddlesnaps/supertuxkart/assets/147548/0bbae08a-b212-43ba-b3d9-6b60022e5bf3)

Reply:
Appreciate the details :pray: 

I messaged folks on Monday, but haven't heard back. Likely everyone is prepping for Riga, which will be a good opportunity to ask these questions :)

Reply:
I might have a fix later today

Reply:
@lucyllewy - do you mind posting the failing build logs?  I was able to build on arm64 via `snapcraft remote-build --build-for arm64` and I'm curious why it failed to build on snapcraft.io